app/controllers/cases/data_request_areas_controller.rb

```diff
@@ -50,6 +56,15 @@ def set_data_request_area
       @data_request_area = @case.data_request_areas.find(params[:id]).decorate
     end
 
+    def set_commissioning_document
+      if @data_request_area.data_requests.exists?
+        @data_request_area.commissioning_document = @data_request_area.build_commissioning_document
+        @commissioning_document = @data_request_area.commissioning_document.decorate
```

Comment:
I find this somewhat confusing. Maybe it would be better to create the commissioning document when the DataAreaRequest is created. 

In the UI it can be hidden until the first data request has been created.